services/121-service/test/helpers/utility.helper.ts

```diff
@@ -113,6 +113,7 @@ export async function removePermissionsFromRole(
   await updatePermissionsOfRole(role.id, {
     label: role.label!,
     permissions: adjustedPermissions as PermissionEnum[],
+    description: role.description!,
```

Comment:
@aberonni why is this needed? the method seems to be only about updating permissions, not about updating description (or label for that matter). Can this be dropped or otherwise renamed?

Reply:
That's a good question and a good catch. The properties on `UpdateUserRoleDto` were not marked as optional, which meant you always had to send all properties to be able to update something. I've updated the logic now so that all the properties are optional, and we only update what is sent. That also means that I could remove `label` from this specific call too.